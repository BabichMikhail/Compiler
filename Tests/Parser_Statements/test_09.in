begin 
   	if ( 10 > 20 ) then begin
		;;;; 
		a := 50;
	end	
	else begin
		begin 
		end;
	end;
end.