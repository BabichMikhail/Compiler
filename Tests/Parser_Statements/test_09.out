begin
   if
         20
      >
         10
   then
      begin
            50
         :=
            a
      end
   else
      begin
         begin
         end
      end
end
